src/ui/components/shared/UserSettingsModal/UserSettingsModal.tsx

```diff
@@ -67,13 +68,12 @@ function Personal() {
           <div className="text-gray-500">{email}</div>
         </div>
       </div>
-      <div className="flex flex-row space-x-3 items-center" data-private>
-        <li className="flex flex-row items-center">
-          <label className="space-y-1.5 pr-36 flex-grow cursor-pointer" htmlFor="disableLogRocket">
+      <div className="flex flex-col" data-private>
```

Comment:
Changing the flex direction seems like an odd change, what's up here?

Reply:
Wait nevermind, there was a reason. Without `flex-col`, the row doesn't occupy the entire width:

![image](https://user-images.githubusercontent.com/15959269/137042853-8c709bf6-33cf-4a82-b5c8-410bcfe3218b.png)